docs/topics/api-references.topic

```diff
@@ -0,0 +1,74 @@
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/"><b>Coroutines (kotlinx.coroutines)</b></a>
+            <br/>
+            <p>A powerful library for asynchronous programming using Kotlin coroutines. It offers high-level coroutine builders, concurrency primitives, structured concurrency, and integration with reactive libraries and frameworks.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/Kotlin/kotlinx.coroutines">kotlinx.coroutines</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-core/"><b>Serialization (kotlinx.serialization)</b></a>
+            <br/>
+            <p>A multiplatform JSON and binary serialization library. It provides a type-safe, compile-time mechanism for converting Kotlin objects to and from a variety of formats such as JSON, CBOR, and Protocol Buffers.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/Kotlin/kotlinx.serialization">kotlinx.serialization</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx-io/"><b>Kotlin I/O Library (kotlinx-io)</b></a>
+            <br/>
+            <p>A multiplatform library for low-level I/O operations. It defines abstractions for reading from and writing to binary streams and buffers, designed to be efficient and portable across all Kotlin platforms.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/Kotlin/kotlinx-io">kotlinx-io</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx-datetime/"><b>Date and Time (kotlinx-datetime)</b></a>
+            <br/>
+            <p>A multiplatform date and time library built on top of the java.time API for JVM and a native implementation for other platforms. It offers a consistent and concise API for working with time-based values.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/Kotlin/kotlinx-datetime">kotlinx-datetime</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx-metadata-jvm/"><b>JVM Metadata (kotlin-metadata-jvm)</b></a>
+            <br/>
+            <p>A library for reading and writing Kotlin metadata stored in JVM class files. It is primarily used by tools such as annotation processors, static analyzers, and compiler plugins.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/JetBrains/kotlin/tree/master/libraries/kotlinx-metadata">kotlinx-metadata</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlin-gradle-plugin/"><b>Kotlin Gradle Plugins (kotlin-gradle-plugin)</b></a>
```

Comment:
The link points to KGP API, not to a reference listing support Gradle plugins.
Someone from Kotlin Built Tools Team should take a look at this sections.

Reply:
Checked with Yahor and updated the text!